services/responses.test.js

```diff
@@ -327,4 +336,123 @@ describe(_startCase(filename), function () {
       fn()(req, mockRes);
     });
   });
+
+  describe('getRouteFromDB', function () {
+    const fn = lib[this.description];
+
+    afterEach(function (done) {
+      return fakeDb.clearMem().then(done);
```

Comment:
Is `then` needed here? I don't see it in this line so maybe we could make them look the same https://github.com/clay/amphora-auth/pull/14/files#diff-fde8b926a5441888c2c16db0c4d1046bR308